straxen/plugins/acqmon_processing.py

```diff
@@ -106,7 +116,13 @@ def compute(self, raw_records_aqmon):
         strax.baseline(records, baseline_samples=self.baseline_samples_aqmon, flip=True)
         aqmon_hits = self.find_aqmon_hits_per_channel(records)
         aqmon_hits = strax.sort_by_time(aqmon_hits)
-        return aqmon_hits
+
+        # in busy runs veto starts get split at chunk edges into two. we introduce hitlets:
+        aqmon_hitlets = strax.create_hitlets_from_hits(aqmon_hits, (0,0), [0, 1000])
+        to_pe = np.ones(808)  # stay in ADC units: these are NIM signals
+        aqmon_hitlets = strax.get_hitlets_data(aqmon_hitlets, records, to_pe=to_pe)
```

Comment:
This is where the current tests are failing. It should actually work like this. When strax.get_hitlets_data is called it lastly complains at https://github.com/AxFoundation/strax/blob/156254287c2037876a7040460b3551d590bf5589/strax/processing/hitlets.py#L214 . My hunch is that we pass something that look's a little different to what https://github.com/AxFoundation/strax/blob/156254287c2037876a7040460b3551d590bf5589/strax/processing/hitlets.py#L222 usually sees and the cached compiled function doesn't like it and seg faults. Any idea on how to fix this? If we delete the cache, we might have the same problem the other way around, no? Maybe we find out what's the difference it doesn't like and provide what it likes.

Reply:
We're back to this issue now:

[...]

Fatal Python error: Segmentation fault
Current thread 0x00007fd8cd53f700 (most recent call first):
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/strax/processing/hitlets.py", line 214 in get_hitlets_data
  File "/home/runner/work/***/***/***/plugins/acqmon_processing.py", line 117 in compute
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/strax/plugin.py", line 596 in do_compute
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/strax/plugin.py", line 488 in iter
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/strax/mailbox.py", line 281 in _send_from
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/threading.py", line 870 in run
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/threading.py", line 932 in _bootstrap_inner
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/threading.py", line 890 in _bootstrap
  
  
 [...]
 
 
  /home/runner/work/_temp/a65694df-fcc9-4870-821e-663ce8843c36.sh: line 1:  2489 Segmentation fault      (core dumped) coverage run --source=*** -m pytest --durations 0
tests/test_daq_reader.py 
Error: Process completed with exit code 139.